chainer/functions/pooling/roi_average_align_2d.py

```diff
@@ -200,12 +202,21 @@ def forward_cpu(self, inputs):
                     w1, w2, w3, w4 = _get_bilinear_interp_params(
                         y, x, y_low, x_low, y_high, x_high)
 
-                    v1 = bottom_data[roi_batch_ind, c, y_low, x_low]
-                    v2 = bottom_data[roi_batch_ind, c, y_low, x_high]
-                    v3 = bottom_data[roi_batch_ind, c, y_high, x_low]
-                    v4 = bottom_data[roi_batch_ind, c, y_high, x_high]
+                    if w1 > 0 and y_low >= 0 and x_low >= 0:
```

Comment:
Do we need `w1 > 0` ?

Reply:
we can skip this when `w1 = 0`

Reply:
If `w1 = 0`, the multiplied value is equals to 0. Does it improve performance?

Reply:
Yes. Since skipping and adding `0` are same, I wonder if comparing with `0` is more efficient than adding `0`. 
But this is a question and not important in this PR.